src/VisualStudio/Core/Def/Implementation/Preview/PreviewUpdater.PreviewDialogWorkspace.cs

```diff
@@ -25,25 +25,37 @@ public void UnregisterTextContainer(SourceTextContainer container)
 
             public void CloseDocument(TextDocument document, SourceText text)
             {
-                if (document is Document)
+                switch (document)
                 {
-                    OnDocumentClosed(document.Id, new PreviewTextLoader(text));
-                }
-                else
-                {
-                    OnAdditionalDocumentClosed(document.Id, new PreviewTextLoader(text));
+                    case Document _:
+                        OnDocumentClosed(document.Id, new PreviewTextLoader(text));
+                        break;
+
+                    case AnalyzerConfigDocument _:
+                        OnAnalyzerConfigDocumentClosed(document.Id, new PreviewTextLoader(text));
+                        break;
+
+                    default:
+                        OnAdditionalDocumentClosed(document.Id, new PreviewTextLoader(text));
```

Comment:
same comment from previous PR about defaults and additional docs.  personally, i don't even like what we had before.  but at least it was only "real doc + additional doc".  now that we're adding more types, having this sort of 'implicit additional default' feels very wrong to me.